CITATION.cff

```diff
@@ -0,0 +1,181 @@
+    family-names: Vinca
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-3051-178X'
+  - given-names: Gamze
+    family-names: ' Ünlü'
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0003-0080-7122'
+  - given-names: Miguel Poblete
+    family-names: Cazenave
+    affiliation: IIASA
+  - given-names: Jarmo S.
+    family-names: Kikstra
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0001-9405-1228'
+  - given-names: Fridolin
+    family-names: Glatter
+    affiliation: IIASA
+  - given-names: Jihoon
+    family-names: Min
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-0020-1174'
+  - given-names: Jan
+    family-names: Steinhauser
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-5989-6855'
+  - given-names: Clara
+    family-names: Orthofer
+    orcid: 'https://orcid.org/0000-0002-7049-7791'
+  - given-names: Thomas
+    family-names: Zipperle
+    orcid: 'https://orcid.org/0000-0003-2512-9508'
+  - given-names: 'Nikolay '
+    family-names: Kushin
+  - given-names: Alessio
+    family-names: Mastrucci
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0002-5611-7780'
+  - given-names: Michael
+    family-names: Pimmer
+  - given-names: Peter
+    family-names: Kolp
+    affiliation: IIASA
+    orcid: 'https://orcid.org/0000-0003-0122-2839'
+identifiers:
+  - type: url
+    value: 'https://github.com/iiasa/message_ix'
+    description: Main repository of hte MESSAGEix framework on Github
+  - type: doi
+    value: 10.5281/zenodo.7006303
+    description: Zenodo entry
+repository-code: 'https://github.com/iiasa/message_ix'
+url: 'https://docs.messageix.org/en/stable/index.html'
+abstract: >-
+  MESSAGEix is a versatile, dynamic, model framework for
+  energy-engineering-economy-environment (E4) systems
+  research.
+
+
+  MESSAGE (without “…ix”) is a specific formulation of a
+  generic linear programming (LP) optimization model for
+  strategic energy planning and integrated assessment of E4
+  systems, developed by the IIASA Energy, Climate, and
+  Environment (ECE) Program since the 1980s. To incorporate
+  feedback between prices and demand levels for energy and
+  commodities, the LP model can optionally be linked to the
+  economic general equilibrium (GE) MACRO model.
+
+
+  The message_ix Python package—also fully usable from
+  R—includes:
+
+      Implementations of MESSAGE, MACRO, and their linkage, in GAMS,
+      Application programming interfaces (APIs) and tools for model building and scientific programming,
+      Extensive documentation and a complete test suite.
+
+  The framework is built on IIASA's ix modeling platform
+  (ixmp), which provides data warehouse features for
+  high-powered numerical scenario analysis.
+keywords:
+  - integrated assessment
+  - energy systems
+  - macro-energy
+  - modelling
+  - scenario analysis
+  - data visualisation
+  - Python package
+license: Apache-2.0
+commit: 139c832a9fc195de97e2b4b911461fb49b6fa455
```

Comment:
What does the commit attribute do? Does this get updated with every release then?

Reply:
Not automatically, no. So I'm not sure it's worth the effort. On the other hand, I thought the Horizon people might find it neat and maybe it won't hurt if it's a bit outdated occasionally afterwards.

Reply:
I'd rather remove this. We already have a clear version-and-release-management, this doesn't need to be duplicated (and maybe create inconsistencies) in the citation.cff

Reply:
Okay, will do :)